Makefile

```diff
@@ -154,6 +154,23 @@ ifeq ($(IMG), $(DEFAULT_IMG))
 endif
 	podman push --tls-verify=${VERIFY_TLS} ${IMG}
 
+# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
+# architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
+# - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
+# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
```

Comment:
I believe for docker-buildx to work, the dockerfile needs
```
GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH}
```
https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/Dockerfile#L24